flink-runtime/src/main/java/org/apache/flink/runtime/jobgraph/tasks/CheckpointCommittingOperator.java

```diff
@@ -18,7 +18,10 @@
 
 package org.apache.flink.runtime.jobgraph.tasks;
 
+import org.apache.flink.runtime.state.StateHandle;
+import org.apache.flink.runtime.util.SerializedValue;
+
 public interface CheckpointCommittingOperator {
 	
-	void confirmCheckpoint(long checkpointId, long timestamp) throws Exception;
+	void confirmCheckpoint(long checkpointId, SerializedValue<StateHandle<?>> state) throws Exception;
 }
```

Comment:
Good point, we could potentially omit it but we kept it for now since it will make testing easier to associate confirmations with specific checkpoints at least.